LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/UdpServer.cs

```diff
@@ -1,413 +0,0 @@
-// Copyright (c) Microsoft. All rights reserved.
-// Licensed under the MIT license. See LICENSE file in the project root for full license information.
-
-namespace LoRaWan.NetworkServer
-{
-    using System;
-    using System.Globalization;
-    using System.Net;
-    using System.Net.Sockets;
-    using System.Security.Cryptography;
-    using System.Text;
-    using System.Threading;
-    using System.Threading.Tasks;
-    using LoRaTools;
-    using LoRaTools.ADR;
-    using LoRaTools.CommonAPI;
-    using LoRaTools.LoRaPhysical;
-    using LoRaTools.Utils;
-    using LoRaWan.NetworkServer.ADR;
-    using Microsoft.Azure.Devices.Client;
-    using Microsoft.Azure.Devices.Shared;
-    using Microsoft.Extensions.Caching.Memory;
-    using Microsoft.Extensions.Logging;
-    using Newtonsoft.Json;
-
-    /// <summary>
-    /// Defines udp Server communicating with packet forwarder.
-    /// </summary>
-    public sealed class UdpServer : IPacketForwarder, IDisposable
-    {
-        private const int PORT = 1680;
-        private readonly NetworkServerConfiguration configuration;
-        private readonly MessageDispatcher messageDispatcher;
-        private readonly LoRaDeviceAPIServiceBase loRaDeviceAPIService;
-        private readonly ILoRaDeviceRegistry loRaDeviceRegistry;
-        private readonly RandomNumberGenerator RndKeysGenerator = new RNGCryptoServiceProvider();
-
-        private IClassCDeviceMessageSender classCMessageSender;
-        private ModuleClient ioTHubModuleClient;
-        private volatile int pullAckRemoteLoRaAggregatorPort;
-        private volatile string pullAckRemoteLoRaAddress;
-        private UdpClient udpClient;
-
-        private Task<byte[]> GetTokenAsync()
-        {
-            var token = new byte[2];
-            this.RndKeysGenerator.GetBytes(token);
-            return Task.FromResult(token);
-        }
-
-        // Creates a new instance of UdpServer
-        public static UdpServer Create()
-        {
-            var configuration = NetworkServerConfiguration.CreateFromEnvironmentVariables();
-
-            var loRaDeviceAPIService = new LoRaDeviceAPIService(configuration, new ServiceFacadeHttpClientProvider(configuration, ApiVersion.LatestVersion));
-            var frameCounterStrategyProvider = new LoRaDeviceFrameCounterUpdateStrategyProvider(configuration, loRaDeviceAPIService);
-            var deduplicationStrategyFactory = new DeduplicationStrategyFactory(loRaDeviceAPIService);
```

Comment:
This class seems to be unused as well now (or only used from tests). Can we make sure that all the orphaned classes are removed as well? Or would we otherwise need to bring them back later?